examples/start_here/chatbot_example.py

```diff
@@ -65,12 +65,14 @@ def outputs2words(tensors, vocab):
     tensors=[loss, src, outputs_inf, tgt], print_func=lambda x: outputs2words(x, dl.voc.index2word),
 )
 
-num_epochs = 1
-logging.info(f"Training only for {num_epochs}. Train longer (~10-20) for convergence.")
+# num_epochs = 1
+max_steps = 50
+# logging.info(f"Training only for {num_epochs}. Train longer (~10-20) for convergence.")
+logging.info(f"Training only for {max_steps} steps. Train longer (~10-20) for convergence.")
```

Comment:
Can you change `Train longer (~10-20) for convergence` to `Train longer for convergence. Recommended 10-20 epochs`? Or something similar so that it is clear that it means 10~20 epochs not steps?

Reply:
ok